packages/server/graphql/rootSchema.ts

```diff
@@ -73,6 +73,9 @@ const withNestedSchema = mergeSchemas({
   typeDefs: `
     type _xGitHubIssue implements TaskIntegration
     type _xGitHubRepository implements RepoIntegration
+    extend type _xGitHubRepository {
```

Comment:
-2 this would be SO COOL, but unfortunately nest-graphql-endpoint isn't smart enough to remove `service` from the request to github, so it requests that field from github & github immediately returns an error (you can reproduce it by integrating github on a team, creating a new team & then trying to push that task to github).

Reply:
I could not reproduce it like you described. I got an unrelated error which was fixed after merging in master. Right now  we're not querying the nested schema directly, so the resolver in line 89 is called and everything is fine. I prepared a fix  https://github.com/ParabolInc/nest-graphql-endpoint/pull/8 so I can keep this field.